ruby/password_resets_controller.rb

```
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+class PasswordResetsController < ApplicationController
+  skip_before_action :authenticated
+
+  def reset_password
+    user = Marshal.load(Base64.decode64(params[:user])) unless params[:user].nil?
```

RuboCop (1.80.2)

[convention] Avoid using `Marshal.load`.

(Security/MarshalLoad)